HELSSA_AGENTS/notification_system/app_code/cores/text_processor.py

```
@@ -0,0 +1,366 @@
+        Args:
+            text: Input text to process
+            context: Additional context information
+            user_type: Type of user (patient/doctor)
+            
+        Returns:
+            TextProcessingResult with processed information
+        """
+        start_time = time.time()
+        
+        try:
+            self.logger.info(f"Starting medical text processing for {APP_NAME}")
```

Ruff (0.12.2)

Undefined name `APP_NAME`

(F821)